CoverMyDotNet.Samples/Program.cs

```diff
@@ -52,21 +64,166 @@ public static void MultipleRequests(Client client)
 					FirstName = "Tom",
 					LastName = "test2",
 					DateOfBirth = DateTime.Parse("4/4/1972")
+				},
+				Payer = new PayerAttributes()
+				{
+					Bin = "773836",
+					PCN = "MOCKPBM",
+					GroupId = "ABC1",
 				}
 			}).Tokens.First().Id);
 			var requests = client.GetRequests(tokens.ToArray());
 			foreach(var v in requests.Requests)
 			{
-				Console.WriteLine("Patient: {0} {1}", v.Patient.FirstName, v.Patient.LastName);
+				Console.WriteLine("Token: {0} {1}", v.Id, v.Tokens.First().Id);
+			}
+			return requests.Requests;
+		}
+
+		public static void DrugSearch(Client client)
+		{
+			var drugResults = client.SearchDrugs("ambien");
+			Console.WriteLine(drugResults.Drugs.Count.ToString() + " Results");
+			foreach(var v in drugResults.Drugs)
+				Console.WriteLine(v.FullName + "  " + v.Id);
+		}
+
+		public static void GetDrug(Client client)
+		{
+			var drug = client.GetDrug("018058");
+			Console.WriteLine(drug.FullName + " comes in the form of " + drug.DosageForm);
+		}
+
+		public static void GetForm(Client client)
+		{
+			var form = client.GetForm("medco_general_pa_form_21039");
+			Console.WriteLine(form.Id + " " + form.Description);
+		}
+
+		public static void SearchForms(Client client)
+		{
+			var forms = client.SearchForms("064691", "OH", "bcbs");
+			Console.WriteLine(forms.Forms.Count + " Results");
+			Console.WriteLine("First form has a score of " + forms.Forms.First().Score);
+		}
+
+		public static void PostTokens(Client client, string[] tokens)
+		{
+			var result = client.PostTokens(tokens);
+			foreach(var v in result.Tokens)
+			{
+				Console.WriteLine(v.Id + " : " + v.RequestId);
+				Console.WriteLine(client.DeleteToken(v.Id).Content);
 			}
 		}
+
+		public static void PostIndicator(Client client)
+		{
+			var indicator = client.PostIndicator(new IndicatorAttributes()
+			{
+				Patient = new PatientAttributes()
+				{
+					FirstName = "Sally",
+					LastName = "test4",
+					DateOfBirth = DateTime.Parse("3/4/1990"),
+					Email = "example@example.com",
+					Address = new AddressAttributes()
+				},
+				Payer = new PayerAttributes()
+				{
+					Bin = "773836",
+					PCN = "MOCKPBM",
+					GroupId = "ABC1",
+				},				
+				Prescriber = new IndicatorPrescriberAttributes()
+				{
+					NPI = "1234567890",		
+					LastName = "Dr. Jones",
+					Address = new AddressAttributes()						
+				},
+				Pharmacy = new IndicatorPharmacyAttributes()
+				{
+					NPI = "1234567890",
+					Address = new AddressAttributes()
+				},
+				Prescription = new IndicatorPrescriptionAttributes()
```

Comment:
I think in order to demonstrate the difference in /indicators get vs search, it would be helpful to include multiple prescriptions here.  https://developers.covermymeds.com/ehr-api.html#post-indicators-search